pyaedt/hfss.py

```diff
@@ -6217,39 +6254,39 @@ def wave_port(
          :class:`pyaedt.modeler.elements3d.FacePrimitive`
             Main object for port creation or starting object for the integration line.
         reference : int, str, list or :class:`pyaedt.modeler.cad.object3d.Object3d`
-            Ending object for the integration line or reference for Terminal solution. Can be multiple objects.
+            The ending object for the integration line or reference for Terminal solution. Can be multiple objects.
         create_port_sheet : bool, optional
-            Whether to create a port sheet or use given start_object as port shee.
+            Whether to create a port sheet or use the start object as the surface
+            to create the port. Default is ``False``.
         create_pec_cap : bool, False
-            Whether to create a port cap or not.
+            Whether to create a port cap or not. Default is ``False``.
```

Comment:
```suggestion
            Whether to create a port cap or not. The default is ``False``.
```
Maxime said that he'd discuss with the team whether to move toward putting the default value after the data type as you have done for this parameter. If we do this, we don't need to put the "The default is ...." sentence. Until a decision is made, I'd like to keep this description consistent with the others.